fe/fe-core/src/main/java/org/apache/doris/datasource/HMSExternalCatalog.java

```diff
@@ -84,10 +83,21 @@ private void init() {
             return;
         }
         for (String dbName : allDatabases) {
-            long dbId = Env.getCurrentEnv().getNextId();
-            dbNameToId.put(dbName, dbId);
-            idToDb.put(dbId, new HMSExternalDatabase(this, dbId, dbName));
+            long dbId;
```

Comment:
Can you also modify `EsExternalCatalog` init logic?

Reply:
How about in next pr?

Reply:
I will modify `EsExternalCatalog` in next commit.